include/uv.h

```diff
@@ -274,6 +274,9 @@ UV_EXTERN void uv_loop_delete(uv_loop_t*);
 UV_EXTERN size_t uv_loop_size(void);
 UV_EXTERN int uv_loop_alive(const uv_loop_t* loop);
 UV_EXTERN int uv_loop_configure(uv_loop_t* loop, uv_loop_option option, ...);
+#ifndef _WIN32
+UV_EXTERN int uv_loop_fork(uv_loop_t*);
```

Comment:
To me, `atfork` implies that something will happen at *the next* fork, ala `pthread_atfork`. That seems misleading.

I went with `uv_loop_fork` for consistency with libev, but if we want something more specific, I  would propose `uv_loop_did_fork` or `uv_loop_after_fork`?